static/js/people.js

```diff
@@ -155,7 +160,9 @@ exports.huddle_string = function (message) {
 };
 
 exports.user_ids_string_to_emails_string = function (user_ids_string) {
-    const user_ids = user_ids_string.split(',');
+    let user_ids = user_ids_string.split(',');
+    user_ids = map_int(user_ids);
```

Comment:
Prefer `const user_ids = map_int(user_ids_string.split(','))`—the type-shifting mutation is unnecessary, and TypeScript won’t like it when it becomes TypeScript.

Reply:
fixed